### ISSUE TITLE
How would I ensure Eclipse Che can run effieicently across all nodes of a cluster?

### ISSUE THREAD

**Issue by willkara** [eclipse/che#22367]
### Summary

I'm in the process of setting up multiple nodes in a cluster. Eclipse Che is the perfect fit for a shared, distributed IDE for my team and I want to be able to make the most use out of it.

What is the best way to ensure I can run Eclipse Che and workspaces on every node? Essentially making sure the gateway and workspaces can make the most use of the resources on the lab?

Would it be ensuring there's an instance of the che-gateway and server are running on each node, or something like a single gateway handling all servers?

I drew something up to help explain my idea:
![Untitled Diagram drawio(1)](https://github.com/eclipse/che/assets/3278887/15ebdb29-df55-4300-8426-e9efafbeffaa)

Thanks!

### Relevant information

TL;DR I have a lot of servers, I want to ensure Eclipse Che and its workspaces can use all of the resources.

**Comment by l0rd** [comment]
This is a scheduler problem rather than a Che problem: when a developer requests a workspace, Che creates a Kubernetes Pod. The Kubernetes scheduler then decides in which node the workspace will be executed. My only recommendation would be to specify carefully, in the devfile that defines the development environments, the memory and CPU requests. This helps the scheduler to take the right decision.

**Comment by willkara** [comment]
Thought something like that would be the issue.

Is there a way to ensure all nodes are available for Che? Or is that
enabled by default?

I guess I was hoping there'd be a more explicit config setting for the
pod/deployment scheduler. I'll experiment a bit, thanks!
